packages/app-store/_appRegistry.ts

```diff
@@ -47,3 +49,31 @@ export async function getAppRegistry() {
   }
   return apps;
 }
+
+export async function getAppRegistryWithCredentials(userId: number) {
```

Comment:
In order to simplify things, I added this util to get apps with the corresponding user credentials for each, letting me show useful information from the apps page as required by the design